.github/workflows/build_debian_package.yml

```
@@ -0,0 +1,90 @@
+name: Build debian package
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
```

```suggestion
        sudo apt-get install --no-install-recommends -y \
```
By default it installs a lot of bloat that we don't need and likely it's better to manual control what packages we have.